image_services/egis/make_egis_services.py

```diff
@@ -174,7 +174,7 @@ def main():
     parser.add_argument('config_file')
     args = parser.parse_args()
 
-    bucket = 'sentinel-1-global-coherence-earthbigdata'
+    bucket = 'asf-ngap2w-p-s1-global-coherence'
     overview_path = '/vsis3/asf-gis-services/public/GSSICB/'
```

Comment:
Are we still going to store overviews in our s3 bucket?

Reply:
Yes, I think that's the best approach for the short term. In a perfect world all data related to the image service (both the geotiffs and the overviews) would be stored in Earthdata Cloud, but I don't see a pressing need to move where the overviews are stored right now.

That does mean we'll have to use one set of AWS keys when reading geotiff data from the EDC bucket (building the csv, creating the mosaic, creating the overview) and a separate set of AWS keys when uploading the overview. That probably means some updates to the python script and maybe some new command line parameters.